src/Hint/Extensions.hs

```diff
@@ -304,10 +315,9 @@ used PatternGuards = hasS f
     g [L _ BodyStmt{}] = False
     g _ = True
 used StandaloneDeriving = hasS isDerivD
-used PatternSignatures = hasS isPatTypeSig
+used ScopedTypeVariables = hasS isPatTypeSig
```

Comment:
This is wrong. There was an extension that allowed you to put pattern signatures in, but if GHC lacks it, then just delete this clause. ScopedTypeVariables is a different and more powerful extension which we can't detect presence of absence of.

Reply:
Ok. This clause will be deleted.